src/lib/bootloader/autoyast_converter.rb

```diff
@@ -33,19 +33,26 @@ def import(data)
         bootloader = bootloader_from_data(data)
         return bootloader if bootloader.name == "none"
 
-        import_grub2(data, bootloader)
-        import_grub2efi(data, bootloader)
-        import_stage1(data, bootloader)
-        import_default(data, bootloader.grub_default)
-        import_device_map(data, bootloader)
-        import_password(data, bootloader)
-        # always nil pmbr as autoyast does not support it yet,
-        # so use nil to always use proposed value (bsc#1081967)
-        bootloader.pmbr_action = nil
-        cpu_mitigations = data.global.cpu_mitigations
-        bootloader.cpu_mitigations = CpuMitigations.from_string(cpu_mitigations) if cpu_mitigations
+        if ["grub2", "grub2-efi"].include?(bootloader.name)
+          import_grub2(data, bootloader)
+          import_grub2efi(data, bootloader)
+          import_stage1(data, bootloader)
+          import_default(data, bootloader.grub_default)
+          import_device_map(data, bootloader)
+          import_password(data, bootloader)
+          # always nil pmbr as autoyast does not support it yet,
+          # so use nil to always use proposed value (bsc#1081967)
+          bootloader.pmbr_action = nil
+          cpu_mitigations = data.global.cpu_mitigations
+          if cpu_mitigations
+            bootloader.cpu_mitigations = CpuMitigations.from_string(cpu_mitigations)
+          end
+        end
+        if bootloader.name == "systemd-boot"
+          bootloader.menue_timeout = data.global.timeout
+          bootloader.secure_boot = data.global.secure_boot
```

Comment:
this should be definitively documented in autoyast documentation in SUSE/sle-doc to avoid surprise of users/customers.

Reply:
It is already documented in this PR
https://github.com/SUSE/doc-sle/pull/1509/files#diff-b3b3b12399b1094b281a3537befddb0f672fe039dc546a93f714b333c1d49ab8R60